REQUIRE

```diff
@@ -2,5 +2,6 @@ julia 0.6
 
 JSON
 MbedTLS
-HTTP 0.5.4
-HttpCommon # for deprecations
+HTTP 0.6
+Nullables
+
```

Comment:
There should also be an explicit Compat dependency here, since you're using `Nothing` and the pairs method for `replace` (among other things).

Reply:
Hah, and `using Compat` itself!

Reply:
With my test script in Pkg3 this would have errored locally...